spark/dl/src/main/scala/com/intel/analytics/bigdl/example/loadmodel/AlexNet.scala

```diff
@@ -137,4 +139,33 @@ object AlexNet {
     val loss = LogSoftMax().setName("loss").inputs(fc8)
     Graph(conv1, loss)
   }
+
+  def dnn(classNum: Int, hasDropout : Boolean = true): Module[Float] = {
+    val model = Sequential()
```

Comment:
can we keep the layer name same with the corresponding layer under nn. Then we just need to import com.intel.analytics.nn.dnn._